iscsi/targetd/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Sorry for the delay, I'd like to do a final review, then I'll merge.

I would like to add this to the CI but let's do it in a follow-up PR.

@humblec expressed interest in reviewing the code itself but I'm not sure he has time so again let's do that later.